src/main/java/mmson/Main.java

```diff
@@ -0,0 +1,21 @@
+package mmson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+        MMSonConverter converter = new MMSonConverter();
+        List<Student> students = new ArrayList<>();
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+                students.add( new Student("a", "b", 1, null));
+        Student p = new Student("asd0", "asd10", 20, students);
+        System.out.println(converter.toJson(p));
```

Comment:
Zamiast do maina możnaby to wrzucić do testów i tam wywoływać `converter.ToJson`, tak jak to teraz jest robione.